docs/content/manual/_incoming/2474-image-filtering-by-labels.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Image filtering by labels
+---
+
+* Related issues: [#2474](https://github.com/harvester/harvester/issues/2474) [backport v1.0] [FEATURE] Image filtering by labels
+
+## Category: 
+* Image
+
+## Verification Steps
+1. Upload several images and add related lable
+1. Go to the image list page 
+1. Add filter according to test plan 1
```

Comment:
What test plan is this referring to? I'm a little confused. Is it referring to the case 1 expected results?

Reply:
The test plan is referring to validate the filter image created by given label in different key value pairs in the Harvester VM and Rancher RKE2 cluster provisioning page. 

Appreciated for the detailed check, I have removed the scenarios statement in the expected behavior section.
And update them with straight check point to increase the readability relating to the test steps section 

And double confirm it works on the Hugo preview